history-of-the-quakers-v3/original/26-book-6-chapter-6.adoc

```diff
@@ -306,6 +310,9 @@ than to be over curious in questions of words, ministering to strife and content
 reminding them of the ancient and constant principle and experience of friends,
 that the dispensation of the gospel committed to them, was a
 
+[.embedded-content-document.epistle]
```

Comment:
So this begins quoting the epistle midway through and it looks a little funny when it's embedded so idk if there's anything we can do to make it look better...ellipses maybe?? Idk.

Reply:
Yeah, lets add an ellipses, like this (note the lack of space after it):

```ASCIIDOC
&hellip;spiritual dispensation; in nowise to oppose,
```